docs/writing-docs/skillmaps.md

```diff
@@ -0,0 +1,222 @@
+
+* activityProperty1: ...
+* activityProperty2: ...
+
+### path-id1-activity2
+
+## path-id2
+
+### path-id2-activity1
+
+### path-id2-activity2
+
+### path-id2-activity3
+```
+
+The first-level (`#`) heading is the ID of the skillmap. Under this heading are the map's properties. A skillmap document has one or more _paths_ declared under the second-level (`##`) headings. The path sections contain a set of _activities_ that form the path. The activities for a path are under third-level (`###`) headings.
+
+Each activity section has list of properties. All activities have a `name`, a `description`, and a `type`, along with other properties which are related to the `type` of the section.
+
+```
+### space-cruiser-activity1
+
+* name: Space background
+* description: Create a stellar background image.
+* type: tutorial
+```
+
+### Map properties
+
+The map properties are in the first section of the skillmap document. This starts with a first-level (`#`) heading using the skillmap's name, or its the _map-id_.
+
+* **map-id**: the identification name of the skillmap
+
+```
+# game-maker-guide
+```
+
+Following the heading is a bulleted list of the properties defined for the skillmap.
+
+* **name**: map display name
+* **description**: map description
+* **infoUrl**: ???
+* **backgroundurl**: URL to the background image
+* **bannerurl**: URL to the banner image (displayed in the sidebar when nothing is selected) 
+* **primarycolor**: hex color value for path and locked nodes (slightly translucent)
+* **secondarycolor**: hex color value for unlocked nodes
+* **tertiarycolor**: hex color value for background
+* **highlightcolor**: hex color value for the selected node border and unlocked reward nodes
+* **alternatesources**: ???
+
+A property list from a skillmap in MakeCode Arcade provides an example:
+
+```markdown
+# game-maker-guide
+
+* name: Game Maker Guide
+* description: Level up your game making skills by completing the tutorials in this guide.
+* infoUrl: skillmap/educator-info/int-map-info
+* bannerUrl: /static/skillmap/platformer/activity4.png
+* backgroundurl: /static/skillmap/space/game-maker-guide-background.png
+* primarycolor: #2EA9B0
+* secondarycolor: #F392BD
+* tertiarycolor: #83C252
+* highlightcolor: #FAED28
+* alternatesources: github:https://github.com/microsoft/pxt-skillmap-sample/skillmap.md
+```
+
+The following skillmap has 3 different paths defined. The panel on the right shows the name of the map with a description and a banner image.
+
+![Full skillmap graphic](/static/skillmaps/skillmap.jpg)
+
+### Path properties
+
+The path properties are in the second-level (`##`) headings using the paths's name, or its _path-id_.
+
+* **path-id**: the identification name of the path
+
+```
+## space
+```
+
+After the heading is the path property list.
+
+* **name**: path display name
+* **description**: the path description
+* **completionUrl**: the URL path to a completion document (i.e. the "Certificate of Completion")
```

Comment:
oh this property should be removed, it's been deprecatd in favor of the completion nodes